src/app/pages/create-account/create-account.page.ts

```diff
@@ -20,31 +20,36 @@ export class CreateAccountPage implements OnInit {
   email: '';
 
   isSubmitted = false;
-
+  signupForm: FormGroup;
```

Comment:
Made the form group a property (rather than creating it specifically in the constructor).

Reply:
What's the advantage of doing this?